src/AccessibilityInsights.SharedUx/Dialogs/HotkeyGrabDialog.xaml

```diff
@@ -5,14 +5,13 @@
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
         xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-        xmlns:local="clr-namespace:AccessibilityInsights.SharedUx.Dialogs"
         xmlns:Properties="clr-namespace:AccessibilityInsights.SharedUx.Properties"
         mc:Ignorable="d"
         ResizeMode="NoResize"
         Title="{x:Static Properties:Resources.HotkeyGrabDialogWindowTitle}" Height="162" Width="319" 
         KeyDown="Window_KeyDown" KeyUp="Window_KeyUp" 
         WindowStartupLocation="CenterOwner"
-        ShowInTaskbar="False" Topmost="True"
+        ShowInTaskbar="False" Topmost="True" FocusManager.FocusedElement="{Binding ElementName=tbHotkey}"
```

Comment:
>True [](start = 39, length = 4)

Possibly out of scope for this PR, but it seems like Topmost should inherit the Topmost flag of the app. Current code allows this window to float if the app is non-Topmost then you switch to another app

Reply:
I want to say we intentionally made this dialog always topmost because you're not really supposed to leave it open (it doesn't even have a close button)

Reply:
I've been convinced. Change made.